pkg/controller/scyllacluster/resource.go

```diff
@@ -330,6 +337,8 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 		},
 		Spec: appsv1.StatefulSetSpec{
 			Replicas: pointer.Ptr(r.Members),
+			// Give some time for kube-proxy to notice Pod and Endpoints/EndpointSlices being ready and reconcile its state.
+			MinReadySeconds: int32(kubeProxyEndpointsSyncPeriod.Seconds()),
```

Comment:
It is, Endpoints declare Ready when Pods become Ready, but before we declare ScyllaCluster ready, we need to give some time to kubeproxy to reconcile them into iptables. Without it, traffic test were failing because kubeproxy didn't manage to set up forwarding.

Reply:
Right, it depends on a LB controller. Those that don't report a condition back need minReadySeconds so the endpoint gets "applied", the others can use pod readiness gates and signal back. I think splitting minReadySeconds into a separate PR that can plumb readiness gates through would be great.